docs/database-migrations.md

```diff
@@ -0,0 +1,24 @@
+# Migrating the Database
+
+This is a bit complicated because of how we connect these things and how it's deployed. This document should walk you through most of the sharp edges.
+
+## Where
+
+We deploy into Heroku right now, and heroku doesn't give us a one-off spot for deploying scripts. To that end, there are three apps in Heroku for Jira:
+
+*  jira-integration-staging
+*  jira-integration-migrations
+*  jira-integration-production
+
+You can deploy -staging and -production with chatops, but for now, -migrations isn't involved with Heaven.
+
+So you will need to go to the jira-integration-migrations page, click on the Deploy tab, and then choose "manual deploy" from your branch.
```

Comment:
I'm confused, why do we need to deploy to `-migrations`? Why can't we just deploy to production and run `./node_modules/.bin/sequelize db:migrate`?

Reply:
Unfortunately not, because if you deploy to `-production` you will require production to then restart to be `ssh`-able (this is required every time you re-deploy, it turns off ssh ability).